charts/incubator/teamspeak3/SCALE/ix_values.yaml

```diff
@@ -0,0 +1,19 @@
+##
+# This file contains Values.yaml content that gets added to the output of questions.yaml
+# It's ONLY meant for content that the user is NOT expected to change.
+# Example: Everything under "image" is not included in questions.yaml but is included here.
+##
+
+image:
+  repository: teamspeak
+  tag: 3.13.6
+  pullPolicy: IfNotPresent
+
+
+##
+# Most other defaults are set in questions.yaml
+# For other options please refer to the wiki, default_values.yaml or the common library chart
+##
+env:
+  #TeamSpeak 3 settings
+  TS3SERVER_LICENSE: accept
```

Comment:
Best to include this in values.yaml as a hidden variable

Reply:
how can i put it hidden in values.yaml?

Should I add it first in the questions.yaml?

Reply:
whoops I meant questions.yaml ofc.